tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -278,6 +278,23 @@ def test_flatnonzero(self, xp, dtype):
         return xp.flatnonzero(array)
 
 
+@testing.parameterize(
+    {'array': numpy.random.randint(0, 2, (20,))},
+    {'array': numpy.random.randn(3, 2, 4)},
+    {'array': numpy.empty((0,))},
+    {'array': numpy.empty((0, 2))},
+    {'array': numpy.empty((0, 2, 0))},
+)
+@testing.gpu
+class TestArgwhere(unittest.TestCase):
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_list_equal()
```

Comment:
The return value is not a list of arrays, but a single array.
```suggestion
    @testing.numpy_cupy_array_equal()
```

Reply:
Done